knowledge_repo/app/templates/create-knowledge.html

```diff
@@ -7,15 +7,26 @@
 {% block content %}
 <center>
   <br>
-  <h3> Select format to download a knowledge stub. </h3>
+  <h3> Select from the options below to add a new post </h3>
```

Comment:
maybe: `Select an option below to add a new post`?

Reply:
👍